src/site/content/en/blog/one-line-layouts/index.md

```diff
@@ -0,0 +1,316 @@
+---
+title: Ten modern layouts in one line of CSS
+subhead: This post highlights a few powerful lines of CSS that do some serious heavy lifting and help you build robust modern layouts.
+authors:
+  - una
+description: This post highlights a few powerful lines of CSS that do some serious heavy lifting and help you build robust modern layouts.
+date: 2020-07-02
+hero: hero.png
+alt: Holy grail layout.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - css
+---
+
+-- video embed --
+
+The evolution of CSS layouts enable developers to write really meaningful and robust styling rules with just a few keystrokes. The talk above and this subsequent post examine 10 powerful lines of CSS that do some serious heavy lifting.
```

Comment:
```suggestion
Modern CSS layouts enable developers to write really meaningful and robust styling rules with just a few keystrokes. The talk above and this subsequent post examine 10 powerful lines of CSS that do some serious heavy lifting.
```
* "The evolution of CSS layouts enable developers to write really meaningful and robust styling rules" doesn't really make sense. It's not the **evolution** of CSS that has enabled this (using the literal definition of evolution as something changing over time). I think what we're actually trying to say here is "modern CSS"
* There's a subject/verb mismatch. The subject is the **evolution** of CSS layouts" (singular) but we're using a plural verb ("enable"). If you want to keep "evolution" then it should be "The evolution of CSS layouts **enables**…"